code/addons/docs/package.json

```diff
@@ -85,7 +85,7 @@
   },
   "dependencies": {
     "@babel/plugin-transform-react-jsx": "^7.19.0",
-    "@jest/transform": "^28.0.0",
+    "@jest/transform": "^29.3.1",
```

Comment:
@kasperpeulen I see that you updated @jest/transform here and in storyshots.  Was that intentional?  This is a dependency that is used by addon-docs, and my storybook is now crashing with an error message coming from `jest-util` via `@jest/transform`.

```
Uncaught TypeError: Cannot read properties of undefined (reading 'isTTY')
    at ./node_modules/jest-util/build/isInteractive.js 
```

Seems to come from https://github.com/facebook/jest/blob/e7edb75f8357090714213f252c5aaaa9b3c29f5f/packages/jest-util/src/isInteractive.ts#L10.  We can't run that in the browser, since `process` isn't defined there.

Reply:
should be trivial to replace `process` with a fake object for browser usage, no?

Reply:
I'm sure we can deal with it, yes. I just like to keep user facing changes separate from internal test updates. 

@SimenB since you're here, is this something that changed from 28 to 29?  I took a quick look and couldn't immediately figure out why this would start to fail now in 29.  Maybe you know offhand?

Reply:
Hm, I upgraded to the latest beta (beta.8), and I no longer get this error.  I guess we can just keep our eyes out for it again.

Reply:
Nothing comes to mind, that code has been the same for years